### DISCUSSION TITLE
unstable_cache errors

### DISCUSSION THREAD

**Discussion by jrwpatterson** [vercel/next.js#59048] [Ideas]
### Goals

1. Option to make sure the unstable_cache does not store the output from an error call

### Non-Goals

1.
2.
3.

### Background

Unstable_cache is great and on a website we are readying for production to be built in the next 3-4 months is working perfectly. But when an error happens on the backend that error is being cached for the long term meaning that without blowing away all cache and knowing that the error is there, all subsequent visits will hit the cached error. 

### Proposal

An option to turn on/off the caching of errors and ignoring cached errors and immediately going and revalidating any cache that errored out, plus the pages that called the cached end point.

**Comment by devRabbani** [comment]
Yeah same problem. Any status? @jrwpatterson
